tests/migrator/get_callable_from_step_test.php

```diff
@@ -23,6 +23,8 @@ protected function setUp(): void
 		$db = $this->new_dbal();
 		$db_doctrine = $this->new_doctrine_dbal();
 		$factory = new \phpbb\db\tools\factory();
+		$db_tools = $factory->get($db_doctrine);
+		$db_tools->set_table_prefix($table_prefix);
```

Comment:
Do we really need to always add the table prefix now? just wondering :)

Reply:
> Do we really need to always add the table prefix now?

Okay, to summarize: table prefix is used currently in the following db tools methods:
- `schema_create_table()`
- `schema_create_index()`
- `schema_rename_index()`
- `schema_create_unique_index()`
- `schema_index_drop()`
- `add_prefix()`
- `remove_prefix()`

and also in the new migration to pass to db tools.
So, may be not :)